conf/env.sh.example

```diff
@@ -0,0 +1,19 @@
+#configuration for all scripts
```

Comment:
While you mention this in the README, you could also mention below that you can set `FLUO_HOME` in env.sh:
```
# Uncomment and set if FLUO_HOME is not set in your .bashrc
#FLUO_HOME=/path/to/fluo-install
```